packages/appstore/src/components/onboarding-new/static-cfd-account-manager.tsx

```diff
@@ -64,8 +64,8 @@ const StaticCFDAccountManager = ({
     is_onboarding_animated,
     is_eu_user,
 }: TStaticCFDAccountManager) => {
-    const icon_size = isMobile() ? 48 : 48;
-    const icon_size_derivx = isMobile() ? 48 : 48;
+    const icon_size = 48;
+    const icon_size_derivx = 48;
```

Comment:
this variable is not using any where, please remove it

Reply:
Since the values of these 2 variables were different in the previous design, I separated them but since they have the same value now, I merge them.